### DISCUSSION TITLE
Large Table Split over multiple A4 pages

### DISCUSSION THREAD

**Discussion by dbareis** [dompdf/dompdf#3385] [Q&A]
I have attached the HTML which displays OK apart from some issue with the images which I haven't looked into.  It is also slow to render but I assume I have to live with that.

The table gets split up into 3 sections rather than filling each A4 page.  I am using v2.0.4 but I also tried the debugger  (which hasn't been updated to support that) so used v2.0.3 in that.  Slightly different output but pretty much the same issue.  I don't know how to read the output.

Any ideas?

Source html & generated PDF:

1. [ActualDumpedHtmlWhichGetsConvertedToPdf.zip](https://github.com/dompdf/dompdf/files/14146116/ActualDumpedHtmlWhichGetsConvertedToPdf.zip)

2. [Generated PDF.zip](https://github.com/dompdf/dompdf/files/14146133/Generated.PDF.zip)

As far as I can see I am not exceeding any of the documented limitations (
[dompdf-README.md at master · dompdf-dompdf · GitHub](https://github.com/dompdf/dompdf/blob/master/README.md))

**Comment by bsweeney** [comment]
I took a quick look and ... I'm perplexed. More investigation is definitely in order.

**Reply by dbareis** [reply]
Thanks, should I submit this as a bug/issue?

**Reply by bsweeney** [reply]
You are welcome to go ahead and open a bug. We can update it as we identify the exact nature of the issue.

**Comment by dbareis** [comment]
Thanks, I have created: https://github.com/dompdf/dompdf/issues/3389
